pages/data/index.vue

```diff
@@ -353,10 +371,34 @@ export default {
       this.$router.replace({ query: { type: firstTabType } })
     } else {
       this.fetchResults()
+      // this.fetchFilters()
     }
   },
 
   methods: {
+    /**
+     * Set active filters based on the query params
+     * @params {Array} filters
+     * @returns {Array}
+     */
+    setActiveFilters: function(filters) {
+      const tags = compose(
+        split(','),
+        toLower,
+        pathOr('', ['query', 'tags'])
+      )(this.$route)
```

Comment:
`(this.$route.query.tags || '').toLowerCase().split(',')`

Reply:
I am pretty sure `query` is always defined.  When there is no query it is an empty object.  Correct me if I'm wrong.

Reply:
You are correct, and this is nice: `(this.$route.query.tags || '')`